client/js/store/modules/auth.js

```diff
@@ -125,7 +127,11 @@ export default {
           }
 
           if (game) {
-            context.dispatch('updateGameThenResetPNumIfNull', game);
+            await context.dispatch('updateGameThenResetPNumIfNull', game);
```

Comment:
Not sure I needed to await this but better to keep things synchronous here.

Reply:
Eh, we'll give it a shot. I guess you'd only await if you were doing async operations inside the action and needed to wait for it to finish.